apps/src/netsim/netsimUtils.js

```diff
@@ -215,6 +237,17 @@ exports.deserializeNumber = function (storedNum) {
 exports.scrubLevelConfiguration_ = function (levelConfig) {
   var scrubbedLevel = _.clone(levelConfig, true);
 
+  // Convert old header spec format to new header spec format
+  scrubbedLevel.routerExpectsPacketHeader =
+      exports.scrubHeaderSpecForBackwardsCompatibility(
+          scrubbedLevel.routerExpectsPacketHeader);
+  scrubbedLevel.clientInitialPacketHeader =
+      exports.scrubHeaderSpecForBackwardsCompatibility(
+          scrubbedLevel.clientInitialPacketHeader);
+
+  // Coerce certain values to string that might have been mistaken for numbers
+  scrubbedLevel.addressFormat = scrubbedLevel.addressFormat.toString();
```

Comment:
Even though `addressFormat` is a text field in Levelbuilder, when you type something like "4.4" there it comes across in the level config as a number.  I need it to be a string, so I coerce it here.